src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,76 @@
-import React from 'react';
+import React, { FormEvent, useState } from 'react';
+import classnames from 'classnames';
+import { client } from '../utils/fetchClient';
+import { Comment } from '../types/Comment';
+
+interface Props {
+  postId: number;
+  onUpdate: (response: Comment) => void;
+}
+
+export const NewCommentForm: React.FC<Props> = ({ postId, onUpdate }) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [text, setText] = useState('');
+  const [hasName, setHasName] = useState(true);
+  const [hasEmail, setHasEmail] = useState(true);
+  const [hasText, setHasText] = useState(true);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
create separate state-object for inputs and for errors - it will be easier to maintain it